cmake/dts.cmake

```diff
@@ -267,6 +270,8 @@ if(SUPPORTS_DTS)
   endif()
   endif(DTC)
 else()
-  file(WRITE ${DEVICETREE_UNFIXED_H} "/* WARNING. THIS FILE IS AUTO-GENERATED. DO NOT MODIFY! */")
-  file(WRITE ${DEVICE_EXTERN_H} "/* WARNING. THIS FILE IS AUTO-GENERATED. DO NOT MODIFY! */")
+  file(WRITE ${DEVICETREE_UNFIXED_H}.new "/* WARNING. THIS FILE IS AUTO-GENERATED. DO NOT MODIFY! */")
+  file(WRITE ${DEVICE_EXTERN_H}.new "/* WARNING. THIS FILE IS AUTO-GENERATED. DO NOT MODIFY! */")
```

Comment:
Let's have a file with the contents
"/* WARNING. THIS FILE IS AUTO-GENERATED. DO NOT MODIFY! */"

in tree so we don't have to create this file all the time.

Reply:
done.